templates/gquorum/docker-compose.yml

```diff
@@ -272,7 +272,7 @@ x-tessera-def:
           }
       EOF
           cat /data/tm/tessera-config-09.json
-          java -Xms128M -Xmx128M -jar /tessera/tessera-app.jar -configfile /data/tm/tessera-config-09.json &> /var/log/tessera/tessera-$$HOSTNAME.log | tee -a /var/log/tessera/tessera-$$HOSTNAME.log
+          /tessera/bin/tessera -configfile /data/tm/tessera-config-09.json &> /var/log/tessera/tessera-$$HOSTNAME.log | tee -a /var/log/tessera/tessera-$$HOSTNAME.log
```

Comment:
Hi @ricardolyn, can you update this one in the `templates/besu/docker-compose.yml` too please so we are consistent? Thank you :)

Reply:
yes, will do. thanks